app/store/message.go

```diff
@@ -138,3 +131,31 @@ func (s *Session) GetMessages(i int) *Message {
 func (m *Message) GetName() string {
 	return TelToName(m.Source)
 }
+
+// FindQuotedMessage searches the equivalent message of DataMessage_Quote in our
+// DB and returns the local message id
+func FindQuotedMessage(quote *signalservice.DataMessage_Quote) (error, int64) {
+	var quotedMessages = []Message{}
+	err := DS.Dbx.Select(&quotedMessages, "SELECT * FROM messages WHERE sentat = ?", quote.GetId())
```

Comment:
You filter on the `sentat` column with an id? Either the parameter is wrongly named or there is a problem here.

Reply:
sentat is a bad name for "timestamp", it's in combination with the "tel"  the unique identifier of a message. The correct way would be to rename it everywhere, but that's in lots places, even in the frontend